docs/pages/en-us/Job-Types.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: default
+title: Job Types
+
+banner:
+    icon: 'assets/img/fork-lift.png'
+    title: Job Service
+    subtitle: Orchestration, Management and Monitoring of Data Processing
+    links:
+        - title: GitHub
+          url: https://github.com/JobService/job-service
+---
+
+# Job Types
+
+Job Service can be configured with job types by including a number of job type definition files.  A new job can specify a job type by omitting the `task` object in the request body, and instead providing a `job` object.  This object includes a `parameters` property, and the job type definition defines how to construct the omitted `task` object from this input.
+
+## Configuring Job Types
+
+To configure Job Service with job types, define the environment variable `CAF_JOB_SERVICE_JOB_TYPE_DEFINITIONS_DIR`.  This is the path to a directory containing job type definition files with the '.yaml' extension.  Changes to these files do not take effect until Job Service is restarted.
+
+For each job type definition, you must also provide some type-specific configuration.  Every job type must be configured with `taskPipe` and `targetPipe` properties, which will be used directly in the constructed `task` object, and a job type definition may specify additional properties (see `configurationProperties` below).  To configure these values, define environment variables using the job type ID (specified in the job type definition) and the property name.  For example, a job type with ID `standard_ingest`, defining the additional property `storeName`, must have the following environment variables defined:
+ 
+ - `CAF_JOB_SERVICE_JOB_TYPE_STANDARD_INGEST_TASK_PIPE`
+ - `CAF_JOB_SERVICE_JOB_TYPE_STANDARD_INGEST_TARGET_PIPE`
+ - `CAF_JOB_SERVICE_JOB_TYPE_STANDARD_INGEST_STORENAME`
+
+## Defining a Job Type
+
+A definition is a YAML file containing an object.  The subsections here correspond to properties of that object.
+
+### id
+
+- required: true
+- type: string
+
+Unique identifier for the job type across all of the Job Service's job types.
```

Comment:
It could work that way; I don't prefer either way.  In the code I treat the definition source independently from its contents, so it doesn't necessarily have a name, but name could be added as a separate concept that also comes from the source.

Reply:
I tend to prefer that the name is used as otherwise the filename and id will get out-of-sync and cause confusion.